assets/components/organisms/restauration/restauration.twig

```diff
@@ -1 +1,104 @@
-🤷‍♀️
+<ul class="nav nav-tabs" role="tablist">
+  {% for day in restauration %}
+    <li class="nav-item">
+      <a class="nav-link {% if loop.first %}active{% endif %}" id="{{ day|lower }}-tab" data-toggle="tab" href="#{{ day|lower }}" role="tab" aria-controls="{{ day|lower }}" {% if loop.first %}aria-selected="true"{% endif %}>{{ day }}</a>
+    </li>
+  {% endfor %}
+</ul>
+
+<div class="tab-content py-3">
+  {% for day in restauration %}
+  <div class="tab-pane fade show {% if loop.first %}active{% endif %}" id="{{ day|lower }}" role="tabpanel" aria-labelledby="{{ day|lower }}-tab">
+    <div class="row my-3">
+      <div class="col-md-4">
+        <select class="select-multiple mb-3" multiple="multiple" data-placeholder="Restaurant">
```

Comment:
Simply include your `select/select-multiple` component insteand of duplicating code 😉